packages/web-components/src/components/tabs-extended/tab.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  html,
+  customElement,
+  property,
+  LitElement,
+  internalProperty,
+} from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './tabs-extended.scss';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The copy content of content item horizontal.
+ *
+ * @element dds-tab
+ */
+@customElement(`${ddsPrefix}-tab`)
+class DDSTab extends StableSelectorMixin(LitElement) {
+  /**
+   * Defines label of the tab.
+   */
+  @property({ reflect: true })
+  label = null;
+
+  /**
+   * Defines title of the tab.
+   */
+  @property({ reflect: true })
+  title = null;
+
+  /**
+   * Defines the disabled state of the tab.
+   */
+  @property({ reflect: true })
+  disabled: Boolean = false;
+
+  /**
+   * Defines the selected state of the tab.
+   */
+  @property({ reflect: true })
+  selected: Boolean = false;
+
+  /**
+   * Defines the index of the tab relative to other tabs.
+   */
+  @internalProperty()
+  private index: Number;
```

Comment:
Prefixing private properties with `_` is our naming convention:

```suggestion
  private _index: Number;
```